src/main/java/com/teamc/fms/entity/TimeSlot.java

```diff
@@ -1,28 +1,33 @@
 package com.teamc.fms.entity;
 
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import jakarta.persistence.*;
 import jakarta.validation.constraints.NotNull;
 import lombok.*;
 
 import java.sql.Time;
 
 @Entity
-@Table(name = "time_slot")
+@Table(name = "timeslot")
 @Getter
 @Setter
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
 public class TimeSlot {
     @Id
+    @SequenceGenerator(name = "timeslot_id_seq", sequenceName = "timeslot_id_seq",
+            allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.AUTO, generator = "timeslot_id_seq")
     private Integer id;
-    @NotNull
+    @NotNull(message = "Start hour cannot be null")
```

Comment:
We do not serialize or deseriaze entities, so annotaions like @JsonIgnore have no meaning here, instead they are used with DTOs
Also no need for the meesages in @NotNull